Code/GraphMol/ForceFieldHelpers/Wrap/rdForceFields.cpp

```diff
@@ -86,6 +86,14 @@ python::object FFConfsHelper(ROMol &mol, ForceFields::PyForceField &ff,
   return pyres;
 }
 
+ForceFields::PyForceField *GetEmptyForceField(ROMol &mol) {
```

Comment:
I think you want this to also take an optional `confId` argument

Reply:
yes, forgott about that, is added